config/z_parameter_sweep_adage.tsv

```diff
@@ -0,0 +1,7 @@
+parameter	sweep
```

Comment:
Can we change sweep to values or something a lil more intuitive?

Reply:
updated to `sweep_values`